docs/source/API/core/view/create_mirror.rst

```diff
@@ -164,4 +164,4 @@ Description
 
      .. important::
 
-	``arg_prop`` must not include a pointer to memory, or a label, or allow padding and ``arg_prop`` must include a memory space.
+	``arg_prop`` must not include a pointer to memory, or a label, or allow padding and ``arg_prop`` must include a memory space. 
```

Comment:
```suggestion
	``arg_prop`` must not include a pointer to memory, or a label, or allow padding and ``arg_prop`` must include a memory space.
```

Reply:
@masterleinad what did you change?

Reply:
oh remove trailing whitespace that was added

Reply:
Not sure why I can't just apply

Reply:
organization fork instead of user...